### DISCUSSION TITLE
Porting this work to LibreHardwareMonitor

### DISCUSSION THREAD

**Discussion by zwamdurkel** [EvanMulawski/FanControl.CorsairLink#214] [General]
Have you considered porting your work to LibreHardwareMonitor so it could be useful in many other scenario's? I would personally love to see it

*Why not do it myself?*
It seems a bit to daunting for my skill level at the moment

**Comment by EvanMulawski** [comment]
Hi @zwamdurkel, that is part of my long-term goal for this project. While it won't be feasible to port it entirely/directly to LHM due to various issues, it would exist in LHM as API calls to a separate service that would need to be installed on the system:

```
LHM <-> Open-Source Corsair Windows Service (future of this project) <-> Corsair hardware
```
